content/efcore/getting-started/samples/loading-related-data/then-include-method/Program.cs

```diff
@@ -12,10 +12,12 @@ public static void Main()
                 .Include(b => b.CheckoutRecords)
                     .ThenInclude(c => c.Reader)
                 .Single(b => b.Id == 1); 
-
+			
+			Console.WriteLine("Book: {0}", book.Title);
+			
             foreach (CheckoutRecord checkoutRecord in book.CheckoutRecords)
             {
-                Console.WriteLine("{0} - {1} {2}", checkoutRecord.Book.Title, checkoutRecord.Reader.FirstName, checkoutRecord.Reader.LastName);
+                Console.WriteLine("Due Date: {0}\n Checked out from Reader: {1} {2}", checkoutRecord.DueDate.ToString("MMMM dd, yyyy"), checkoutRecord.Reader.FirstName, checkoutRecord.Reader.LastName);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.